spec/support/generators.rb

```diff
@@ -0,0 +1,22 @@
+module Pageflow
+  module Specs
+    module Generators
+      module Macros
```

Comment:
Move this file to `spec/support/helpers` and it will get required automatically in the `spec_helper` no need to require it separately then.